### PR TITLE
fix typo in samplers.rst

### PR DESCRIPTION
**Pull request by quackduck**
fixed a typo in docs

### CODE REVIEW

File: docs/overview/samplers.rst [hunk 1]
```diff
@@ -22,7 +22,7 @@ compute resources (:term:`solver`\ s) of different types:
   quadratic models (:term:`DQM`), ``hybrid_discrete_quadratic_model_version<x>``
 * :ref:`using_cpu` such as :class:`~dimod.reference.samplers.ExactSolver` for
   exact solutions to small problems
-* :ref:`using_qpu` such the Advantage and D-Wave 2000Q systems.
+* :ref:`using_qpu` such as the Advantage and D-Wave 2000Q systems.
```
Comment (JoelPasvolsky) [inline docs/overview/samplers.rst:25]: While you're updating, would you mind deleting the now-retired D-Wave 2000Q?
```
* :ref:`using_qpu` such as the Advantage.
```
Thanks!

Comment (quackduck) [reply docs/overview/samplers.rst:25]: sure!

**Review by JoelPasvolsky — APPROVED** [review]
Thank you!

File: docs/overview/samplers.rst [hunk 2]
```diff
@@ -109,7 +109,7 @@ minor-embedding.
   :scale: 50 %
   :alt: View rendered by Ocean's problem inspector.
 
-  View of the logical and embedded problem rendered by Ocean's problem inspector. The AND gate's original BQM is represented on the left; its embedded representation on a D-Wave 2000Q system, on the right, shows a two-qubit chain (qubits 176 and 180) for variable :math:`x2`. The tool is helpful in visualizing the quality of your embedding.
+  View of the logical and embedded problem rendered by Ocean's problem inspector. The AND gate's original BQM is represented on the left; its embedded representation on a D-Wave system, on the right, shows a two-qubit chain (qubits 176 and 180) for variable :math:`x2`. The tool is helpful in visualizing the quality of your embedding.
```
[the remark this answers is not in the document]
Comment (arcondello) [reply docs/overview/samplers.rst:112]: I was assuming we would do that as a followup, because we would have to do it on our side.

**Pull request merged**